server.js

```diff
@@ -42,4 +51,9 @@ app.get('/block-me/server-sent-events', (req, res) => {
 // dummy CSP report endopoint
 app.post('/block-me/csp', (req, res) => {
     return res.sendStatus(200);
+});
+
+// reflects request headers back
+app.get('/reflect-headers', (req, res) => {
+    return res.json({headers: req.headers});
```

Comment:
that's how we get info about headers sent - this endpoint returns them as a JSON file